content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -122,8 +122,8 @@ The default configuration can be printed out using the
 If your configuration is not using the latest version it is **recommended** that you migrate using
 the [kubeadm config migrate](/docs/reference/setup-tools/kubeadm/kubeadm-config/) command.
 
-For more information on the fields and usage of the configuration you can navigate to our API reference
-page and pick a version from [the list](https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#section-directories).
+For more information on the fields and usage of the configuration you can navigate to our
+[API reference page](/docs/reference/config-api/kubeadm-config.v1beta2/).
```

Comment:
is there a way to navigate to a page that lists all kubeadm APIs?

that is what the godoc link is doing:
https://pkg.go.dev/k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm#section-directorie

so given this is for the main branch we should link to v1beta2.
and for the dev-1.22 branch we should link to v1beta3?

Reply:
@tengqm , Does it make sense to link to configuration API index page?
https://kubernetes.io/docs/reference/config-api/

Reply:
@neolit123 @kbhawkey  I'm not sure we have such a use case for linking to both versions. In 1.21 (main), we point users to v1beta2; while in 1.22, we point users to v1beta3. If there IS a strong use case for users to browse through the different versions, we can change the link to the config-api section. That also means we may need a dedicated subsection for kubeadm configs. Make sense?

Reply:
pointing to v1beta3 only seems mostly fine, but we have to keep in mind that it would take 1 year for users to migrate to the new version. having the reference for the old v1beta2 here makes sense.

Reply:
:+1: to 
> having the reference for the old v1beta2 here makes sense.

based on justification above.
(I'm happy to see this change merge, that'd be a separate improvement).